app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Public Recipes</h1>
+<%@recipes.each do |recipe|%>
+<%= render partial: "shared/publicRecipes", locals: { recipe: recipe } %>
+<% end %>
+<%= link_to 'View your Food', user_foods_path(@user) %>
```

Comment:
- [ ] Your app is working fine as expected :+1: However, it will be nice if you could implement a navigation menu that will allow users to navigate around the app. Currently, a user can create, delete and view the details but is unable to return back to the homepage or other pages because there is no navigation menu.